src/layman/layer/util.py

```diff
@@ -76,8 +76,9 @@ def get_providers():
 
 
 def fill_in_partial_info_statuses(info, chain_info):
-    file_type = info.get('file', dict()).get('file_type')
-    item_keys = get_layer_info_keys(file_type) if file_type else settings.FILE_TYPE_UNKNOWN
+    file_type = info.get('file', dict()).get('file_type', settings.FILE_TYPE_UNKNOWN)
```

Comment:
file_type can be UNKNOWN for external tables, that is wrong. We can use something like
`settings.FILE_TYPE_UNKNOWN -> info['_file_type']`